install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -99,66 +104,214 @@ pv0001                   <none>    5368709120   RWO           Available
 ----
 ====
 
-Users can then link:../../dev_guide/persistent_volumes.html[request storage
-using persistent volume claims], which can now utilize your new persistent
-volume.
+The next step can be to create a persistent volume claim (PVC) which will bind
+to the new PV:
 
-[[nfs-enforcing-disk-quotas]]
+.PVC Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-claim1
+spec:
+  accessModes:
+    - ReadWriteOnce <1>
+  resources:
+    requests:
+      storage: 1Gi <2>
+----
+<1> As mentioned above for PVs, the `*accessModes*` do not enforce security, but
+rather act as labels to match a PV to a PVC.
+<2> This claim will look for PVs offering *1Gi* or greater capacity.
+====
+
+Save the definition to a file, for example *_nfs-claim.yaml_*, and create the
+PVC:
+
+====
+----
+# oc create -f nfs-claim.yaml
+----
+====
+
+[NOTE]
+====
+PVs and PVCs are not necessary, just convenient, and make sharing a volume
```

Comment:
Also note that upstream there is an effort to disallow defining the storage plugin directly in the pod spec.
The proposal is to control, via a PSP "allow" setting, the pod's ability to define persistent storage inline.
The goal is to dissuade developers from doing this and direct them to use PVCs.
jeff

----- Original Message -----
> > +rather act as labels to match a PV to a PVC.
> > +<2> This claim will look for PVs offering *1Gi* or greater capacity.
> > +====
> > +
> > +Save the definition to a file, for example *_nfs-claim.yaml_*, and create
> > the
> > +PVC:
> > +
> > +====
> > +----
> > +# oc create -f nfs-claim.yaml
> > +----
> > +====
> > +
> > +[NOTE]
> > +====
> > +PVs and PVCs are not necessary, just convenient, and make sharing a volume
> 
> I've taken it out of a "NOTE" box and rewritten/reworked it into the
> Overview. New intro paragraphs:
> 
> > OpenShift clusters can be provisioned with persistent storage using NFS.
> > Persistent volumes (PVs) and persistent volume claims (PVCs) provide a
> > convenient method for sharing a volume across a project. While the
> > NFS-specific information contained in a PV definition could also be
> > defined directly in a pod definition, doing so does not create the volume
> > as a distinct cluster resource, making the volume more susceptible to
> > conflicts.
> 
> > This topic covers the specifics of using the NFS persistent storage type.
> > Some familiarity with OpenShift and NFS is beneficial. See the Persistent
> > Storage architecture topic for details on the OpenShift persistent volume
> > (PV) framework in general.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-docs/pull/1726/files/9a5c4affe7d76091de47c6b741c73c6702c25c43#r56526089